### PR TITLE
feat: Raspberry Pi dess tutorial

### PR DESCRIPTION
**Pull request by realvarx**
**- What I did**

Created a tutorial for dess on a Raspberry Pi

**- How I did it**

Created it using Markdown

**- How to verify it**

1. Clone this repo
2. Run `npm install` and `npm run start`
3. Navigate to `http://localhost:[port]/docs/tutorials/dess/rpi/`

**- Description for the changelog**

Dess tutorial added for Raspberry Pi

### CODE REVIEW

File: content_dev/docs/tutorials/dess/rpi/7-activation/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: codelab
+
+title: 'Activation of @sign' # Step Name
```
Comment (JeremyTubongbanua) [inline content_dev/docs/tutorials/dess/rpi/7-activation/index.md]: branding
'@sign' --> 'atSign'

File: content_dev/docs/tutorials/dess/rpi/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: codelab-list
+
+title: Setup dess on a Raspberry Pi # Title of the codelab
+lead: # Description of the codelab
+description: | # SEO Description of the codelab
+    My SEO Description
```
Comment (JeremyTubongbanua) [inline content_dev/docs/tutorials/dess/rpi/_index.md]: Add SEO description like

"Learn how to setup dess on a Raspberry PI"

**Review by JeremyTubongbanua — CHANGES_REQUESTED** [review]
This is some insane work @realvarx !!! :)

**Review by xavierchanth — DISMISSED** [review]
Please see Jeremy's requested changes, otherwise, it looks good 👍🏻

File: content_dev/docs/tutorials/dess/rpi/7-activation/index.md (hunk 1 above)
Comment (realvarx) [reply content_dev/docs/tutorials/dess/rpi/7-activation/index.md]: Thank you @JeremyTubongbanua!! I have made the rebranding and SEO description modifications 😃

**Review by JeremyTubongbanua — APPROVED** [review]
LGTM

**Pull request merged**